src/main/java/com/savvato/tribeapp/dto/NotificationDTO.java

```diff
@@ -0,0 +1,51 @@
+package com.savvato.tribeapp.dto;
+
+import java.time.LocalDateTime;
+
+
+public class NotificationDTO {
+    private String description;
```

Comment:
DTO's don't have getters and setters, just public attributes. Look into Lombok's @Builder annotation as well. It will apply here.